test/src/main/scala/scala/test/junit/specs2/Specs2Tests.scala

```diff
@@ -6,10 +6,14 @@ import scala.test.junit.support.JUnitCompileTimeDep
 class JunitSpecs2Test extends SpecWithJUnit {
 
   "specs2 tests" should {
-    "run smoothly in bazel" >> {
```

Comment:
Does the filtering feature still support `>>` or only `in`?

Reply:
Should be supported with `>>` too, but haven't tested yet.

Reply:
Are you planning on adding other tests in a different PR?

Reply:
but now you only have `in` right? How do you know `>>` is supported?

Reply:
I checked, the `>>` is also supported, because it's not a part of the test name, but I'll add a test for it.